andela/lotus-ah
# #160451618 Users should be able to like, dislike and unlike article comments

#### What does this PR do?
This is PR is used to implement the like, dislike and unlike of comment functionality. It is also used to fetch all the liked and disliked of the article comments.
   
#### Description of Task to be completed?
- Like comment
- Dislike comment
- Unlike and Un-dislike comment
- get all likes for a specific comment
- get all dislikes for a specific comment

#### How should this be manually tested?
- Start the app locally on port `3000`
- Like an article via postman with the structure below
- Make a `POST` request to `/api/v1/articles/comments/:commentId/like`,`/api/v1/articles/comments/:commentId/dislike`,`/api/v1/articles/comments/:commentId/unlike`
- You should get a response with the structure below:
```js
{
    "status": "SUCCESS",
    "DBdata": {
        "id": 2,
        "userId": 1,
        "likes": true,
        "dislike": false,
        "articleId": null,
        "updatedAt": "2018-10-23T12:42:35.930Z"
    },
    "message": "you liked the comment"
}
```

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
-[#160451618](https://www.pivotaltracker.com/story/show/160451618)
